rocksdb-sys/build.rs

```diff
@@ -43,9 +43,9 @@ fn main() {
     let target_arch = env::var("CARGO_CFG_TARGET_ARCH")
         .expect("CARGO_CFG_TARGET_ARCH is set by cargo.");
 
-    if target_arch == "arm" || target_arch == "aarch64" {
-        cfg.define("PORTABLE", "ON");
-    }
+    // Added to support old CPUs
+    // see https://github.com/paritytech/parity-ethereum/issues/9684
+    cfg.define("PORTABLE", "ON");
```

Comment:
Maybe we could use an env variable to allow compiling with `PORTABLE=OFF`, so that we distribute portable binaries, but allow compiling from source with all the features for ultimate performance?